app/code/Magento/Shipping/Model/Carrier/AbstractCarrier.php

```diff
@@ -325,6 +325,7 @@ public function checkAvailableShipCountries(\Magento\Framework\DataObject $reque
      *
      * @param \Magento\Framework\DataObject $request
      * @return $this|bool|\Magento\Framework\DataObject
+     * @deprecated
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
     public function proccessAdditionalValidation(\Magento\Framework\DataObject $request)
```

Comment:
This method's phpdoc should just be `{@inheritdoc}`, precisely to avoid duplication.